docs/development/design/new-sphinx-guides.rst

```diff
@@ -111,33 +112,110 @@ highly recommended.
       -  Rebuild the HTML docs ``make html`` and notice that now the
          times are printed (yay!)
 
-5. Writing narrative documentation with Sphinx
+6. Writing narrative documentation with Sphinx
 
    -  First focus on ``index.md``, diving more into Markdown
       and mentioning Semantic Line Breaks.
    -  Then add another ``.md`` file to teach how ``toctree`` works.
    -  Then continue introducing elements of the syntax to add pictures,
       cross-references, and the like.
 
-6. Describing code in Sphinx
+7. Describing code in Sphinx
 
    -  Explain the Python domain as part of narrative documentation to
       interleave code with text, include doctests, and justify the
       usefulness of the next section.
 
-7. Autogenerating documentation from code in Sphinx
-8. Deploying a Sphinx project online
+8. Autogenerating documentation from code in Sphinx
+9. Deploying a Sphinx project online
 
    - A bit of background on the options: GitHub/GitLab Pages,
      custom server, Netlify, Read the Docs
    - Make reference to Read the Docs tutorial
 
-9. Appendix: Using Jupyter notebooks inside Sphinx
+10. Appendix: Using Jupyter notebooks inside Sphinx
+11. Appendix: Where to go from here
+
+   - Refer the user to the Sphinx, reST and MyST references,
+     prominent projects already using Sphinx,
+     compilations of themes and extensions,
+     the development documentation.
 
 .. note 
 
    - Looks like MathJax is enabled by default now? Can't see a reference in the docstrings
 
+Read the Docs tutorial
+----------------------
+
+1. What is Read the Docs
+2. Getting started
+
+   1. Preparing our project on GitHub
+
+      - Fork a starter GitHub repository (notice that
+        even an empty project is ready for build the first time without changes,
+        resulting in a nice "welcome to read the docs" dummy page,
+        but using something like `our demo template
+        <https://github.com/readthedocs/template>`_,
+        as a starting point
+        helps mimicking the `sphinx-quickstart` or `cookiecutter` step
+        without having to checkout the code locally)
+
+   2. Importing our project to Read the Docs
+
+      - Sign up on RTD
+      - Connect your GitHub account (is this step necessary
+        if the user signed up with GitHub?)
+      - Connect your GitHub account (to avoid manual import)
+      - Import the project (don't "Edit advanced project options", we
+        will do this later)
+      - The project is created on RTD
+      - Browse "builds", wait a couple of minutes, open the docs (yay!)
+
+   3. Basic configuration changes
+
+      - Add a description, homepage, and tags
+      - Browse other project with similar tags as yours
+        (this way we encourage tagging projects, as well as discovery)
```

Comment:
This is not a great experience, unfortunately. Not sure if we want to promote it, unless we curate the tags they use -- but that will just show a bunch of test projects haha